readme.md

```diff
@@ -348,6 +348,7 @@ Check out my [blog](https://blog.sindresorhus.com) 🦄 or say *hi* on [Twitter]
 - [documentation.js](http://documentation.js.org) - API documentation generator with support for ES2015+ and flow annotation.
 - [YUIDoc](http://yui.github.com/yuidoc/) - Generates API documentation from comments in source.
 - [ESDoc](https://esdoc.org) - Documentation generator targeting ES2015, attaching test code and measuring documentation coverage.
+- [Docsify](https://docsify.js.org) - Magical markdown documentation site generator with multiple themes, no statically built html files.
```

Comment:
The description should succinctly describe what makes this unique from all the others. Magical doesn't say anything. All of the others supports multiple themes.